scripts/generate_noisy_flores.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import json
+import os
+import pickle
+
+from src.datasets.translation_datasets import FloresDataset
+from src.utils.augmentation import TextAugmentationWrapper, init_augmentations
+from src.utils.common import save_json
+from src.utils.config import AugmentationConfig
+from tqdm import tqdm
+
+
+def add_noise_flores(
+    probas_path: str,
+    leet,
+    clusters,
+    expected_changes_per_word,
+    proba_per_text,
+    expected_changes_per_text,
+    max_augmentations,
+    save_dir,
+    split="dev",
+):
+    assert split in ["dev", "devtest"], "Split for FLORES dataset must be dev or devtest"
+
+    with open(probas_path, "rb") as f:
+        probas = pickle.load(f)
+
+    pairs = list(probas.keys())
+    lang2label: dict = {}
+    label2lang: dict = {}
+    label = 0
+    for pair in pairs:
+        for lang in pair.split("-"):
+            if lang not in lang2label:
+                lang2label[lang] = label
+                label2lang[label] = lang
+                label += 1
```

Comment:
I don't think that we need this complex logic. For Flores dataset we can iterate over all samples, apply augmentations, and save in the same format. So only values for `sentence_{lang}` would change.

This is very convenient for our pipeline -- we can reuse `FloresDataset` class, but specify to load data from disc, not from HF.